From 2d170899c2ad9c3981028f3903045c022431c38d Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Tue, 14 Dec 2021 13:36:41 +0100
Subject: [PATCH] p3636-0001: Increase kernel size on TX2 NX

Set TX2 NX kernel size to 128MiB,
same as for the TX2, in order to avoid kernel
hanging when loading a dtb from the filesystem.
Previously, the filesystem dtb in RAM and the
kernel overlapped, causing the kernel to hang
when loading any dtb trough extlinux.conf.

Upstream-status: Pending
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 include/configs/p3636-0001.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/p3636-0001.h b/include/configs/p3636-0001.h
index 51c7af6a8b..bffca8348c 100644
--- a/include/configs/p3636-0001.h
+++ b/include/configs/p3636-0001.h
@@ -29,7 +29,7 @@
 		"ramdisk_addr_r\0" \
 	"kernel_addr_r_align=00200000\0" \
 	"kernel_addr_r_offset=00080000\0" \
-	"kernel_addr_r_size=02000000\0" \
+	"kernel_addr_r_size=08000000\0" \
 	"kernel_addr_r_aliases=loadaddr\0" \
 	"fdt_addr_r_align=00200000\0" \
 	"fdt_addr_r_offset=00000000\0" \
-- 
2.17.1

